index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> -->
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action="" id="survey-form">
+      <label for="name" id="name-label">Name</label>
+      <input type="text" id="name" name="name" placeholder="Enter Your Name" required></input>
+      <label for="email" id="email-label">Email</label>
+      <input type="email" id="email" name="email" placeholder="Enter your Email" required></input>
+      <label for="number" id="number-label">Age <sup>(Optional)</sup></label>
+      <input type="number" id="number" name="number" min="5" max="35" placeholder="Age"></input>
+      <label for="dropdown">Which option best describes your current role?</label>
+      <select name="" id="dropdown">
+        <option disabled selected>Select an Option</option>
+        <option value="software-developper">Software Developper</option>
+        <option value="engineer">Engineer</option>
+      </select>
+      <label for="">Would you recommend freeCodeCamp to a friend?</label><br>
+      <input type="radio" name="radio" value="value-1">
+      <span>Value - 1</span><br>
+      <input type="radio" name="radio" value="value-2">
+      <span>Value - 2</span><br><br>
+      <label for="">What would you like to see improved? (Check all that apply)</label><br>
+      <input type="checkbox" name="" value="val-1">
+      <span>Value-1</span><br>
+      <input type="checkbox" name="" value="val-2">
+      <span>Value-2</span><br>
+      <input type="checkbox" name="" value="val-3">
+      <span>Value-3</span><br>
+      <input type="checkbox" name="" value="val-4">
+      <span>Value-4</span><br>
+      <input type="checkbox" name="" value="val-5">
+      <span>Value-5</span><br><br>
```

Comment:
Changes required:
Try to complete the "for" and "name" attributes in these two inputs, besides using better "value" values (instead of value-1, value-2). By doing these, you'll complete all the required information in the survey form, in order for the user to understand.